### DISCUSSION TITLE
blog/adding-comments/

### DISCUSSION THREAD

**Discussion by giscus** [gamedolphin/gamedolphin.github.com#2] [Announcements]
# blog/adding-comments/

Sandeep Nambiar's personal website

https://www.sandeepnambiar.com/blog/adding-comments/

**Comment by m00sp** [comment]
Awesome work brother, congratulations. For sure that I'm going to use this on my website: https://luis-ti.dev.br, I use the Abridge theme. Thank you so much and greetings from Brasil. Namaste :pray::pray::pray:.
